docker/.env

```diff
@@ -17,6 +17,7 @@
 
 
 COMPOSE_PROJECT_NAME=superset
+DEV_MODE=true
```

Comment:
Is there anything that needs to be done for official docker images built during releases to remove this flag?